### PR TITLE
refactor: 💡 added gtag & sitemap

### PR DESCRIPTION
**Pull request by roy-shiloh**
added gtag

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] UI changes have been reviewed
- [x] No UI review needed

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -19,6 +19,16 @@ const config = {
 		[
 			'@docusaurus/preset-classic',
 			{
+				sitemap: {
+					changefreq: 'weekly',
+					priority: 0.5,
+					ignorePatterns: ['/tags/**'],
```
Comment (tal-rofe) [inline docusaurus.config.js:25]: change

File: docusaurus.config.js [hunk 2]
```diff
@@ -19,6 +19,16 @@ const config = {
...
+					filename: 'sitemap.xml',
+				},
+				gtag: {
+					trackingID: 'G-GW6NMY47QH',
+					anonymizeIP: true,
```
Comment (tal-rofe) [inline docusaurus.config.js:30]: do you need it or not

**Review by tal-rofe — CHANGES_REQUESTED** [review]
change

File: docusaurus.config.js (hunk 2 above)
Comment (roy-shiloh) [reply docusaurus.config.js:29]: yes

**Review by tal-rofe — APPROVED** [review]

**Pull request merged**